docs/development/testing.md

```diff
@@ -44,16 +45,34 @@ would run `npm run test -- -g ipc`.
 [standard-addons]: https://standardjs.com/#are-there-text-editor-plugins
 
 ### Testing on Windows 10 devices
+
+#### Extra steps to run the unit test:
+
+1. Visual Studio 2019 must be installed.
+2. Node headers have to be compiled for your configuration.
+   ```
```

Comment:
Seems like CI is failing at the lint step because the triple ticks do not specify a language.

```console
docs/development/testing.md
  53:4-55:7  warning  Missing code-language flag  fenced-code-flag  remark-lint
  57:4-61:7  warning  Missing code-language flag  fenced-code-flag  remark-lint
```